modules/admin/admin-ui/src/main/resources/web/admin/common/js/form/set/optionset/FormOptionSetOptionView.ts

```diff
@@ -381,8 +437,11 @@ module api.form {
         private updateViewState() {
             this.expand(this.isOptionSetExpandedByDefault() || this.getThisPropertyFromSelectedOptionsArray() != null);
 
-            if (this.isOptionSetExpandedByDefault() && this.getThisPropertyFromSelectedOptionsArray() == null) {
-                this.disableFormItems();
+            if (this.getThisPropertyFromSelectedOptionsArray() == null) {
+                if (this.isOptionSetExpandedByDefault()) {
+                    this.disableFormItems();
+                }
+                this.cleanValidationForThisOption();
```

Comment:
Can we replace double calls to isOptionSetExpandedByDefault() and getThisPropertyFromSelectedOptionsArray() with local variables? Also replace comparison with null with (!)